x-pack/plugins/rollup/public/crud_app/sections/job_create/job_create.js

```diff
@@ -543,7 +543,7 @@ export class JobCreateUi extends Component {
     }
 
     return (
-      <EuiPageContentBody restrictWidth style={{ width: '100%' }}>
+      <EuiPageBody restrictWidth style={{ width: '100%' }}>
```

Comment:
There's some weird spacing issue now

Before
<img width="1409" alt="Screenshot 2023-10-12 at 12 39 39 PM" src="https://github.com/elastic/kibana/assets/5226211/aa828fff-3a48-43fa-95dc-8f5f6d9b84c3">

After
<img width="1216" alt="Screenshot 2023-10-12 at 12 39 44 PM" src="https://github.com/elastic/kibana/assets/5226211/16a03169-837b-4ed3-9a2c-2edb9e8f25cd">

Reply:
Should be fixed by 624b6e476b788a8739b4f2e7d289ddc5daa4d413